README.md

```diff
@@ -45,35 +45,47 @@ resource "aws_ami_from_instance" "example" {
+| `github_organization`        | ``                                           | GitHub organization name                                                         | Yes      |
+| `github_team`                | ``                                           | GitHub team                                                                      | Yes      |
+| `ansible_playbook`           | ``                                           | Path to the playbook - required for `tf_ansible` (e.g. `./admin_tier.yml`)       | Yes      |
+| `ansible_arguments`          | []                                           | List of ansible arguments (e.g. `["--user=ubuntu"]`)                             | No       |
+| `instance_type`              | `t2.micro`                                   | The type of the creating instance (e.g. `t2.micro`)                              | No       |
+| `vpc_id`                     | ``                                           | The id of the VPC that the creating instance security group belongs to           | Yes      |
+| `security_groups`            | []                                           | List of Security Group IDs allowed to connect to creating instance               | Yes      |
+| `subnets`                    | []                                           | List of VPC Subnet IDs creating instance launched in                             | Yes      |
+| `associate_public_ip_address`| `true`                                       | Associate a public ip address with the creating instance. Boolean value          | No       |
+| `comparison_operator`        | `GreaterThanOrEqualToThreshold`              | Arithmetic operation to use when comparing the specified Statistic and Threshold | Yes      |
+| `metric_name`                | `StatusCheckFailed_Instance`                 | Name for the alarm's associated metric                                           | Yes      |
+| `evaluation_periods`         | `5`                                          | Number of periods over which data is compared to the specified threshold         | Yes      |
+| `metric_namespace`           | `AWS/EC2`                                    | Namespace for the alarm's associated metric                                      | Yes      |
+| `applying_period`            | `60`                                         | Period in seconds over which the specified statistic is applied                  | Yes      |
+| `statistic_level`            | `Maximum`                                    | Statistic to apply to the alarm's associated metric                              | Yes      |
+| `metric_threshold`           | `1`                                          | Value against which the specified statistic is compared                          | Yes      |
+| `default_alarm_action`       |`action/actions/AWS_EC2.InstanceId.Reboot/1.0`|  List of actions to execute when this alarm transitions into an ALARM state      | Yes      |
```

Comment:
https://github.com/cloudposse/tf_instance/pull/3/files#diff-c9ac8098c5ea9d3e6a9a596ff0c512a4R100 is not type list